src/app/researcher/study/request/[memberSlug]/study-proposal-form.tsx

```diff
@@ -31,95 +39,83 @@ export const StudyProposalForm: FC<{
     const { user } = useUser()
 
     return (
-        <Paper p="md">
-            <Stack gap="lg">
-                <Title order={4}>Study Proposal</Title>
-                <Divider />
-                <Text>
-                    This section is here to help you submit your study proposal. Consider providing as much detail as
-                    possible to ensure the Reviewer has all the information needed to make an informed decision.
-                </Text>
+        <Paper pt="sm" pl="xl" pr="lg" pb="lg">
```

Comment:
same thing maybe a simple consistent `p='lg'` or something

Reply:
or alternatively like we talked about using the stack gap property to align things a little if possible 🤔 this is me just nitpicking though